### ISSUE TITLE
Holo dialogs?!?!

### ISSUE THREAD

**Issue by devycarol** [devycarol/Emilla#2]
Yup, 4.4 KitKat. What on earth is going on here!?

Modern AlertDialogs are ludicrous. The touch targets for the action buttons on yes/no dialogs are in flagrant violation of Google's own accessibility guidelines, and the lack of dividers in list dialogs make them very difficult to parse where touch-bounds start and end. Don't even get me started on that horrid time picker or whatever atrocities they're committing against corners in Material 3.

Thus, I'm using the most recent dialog style that had a comfortable layout—way back from the Holo days. The hope is to replace this antequated dialog style with a custom style for the app. The plan is to have the exact same shape/layout as the holo dialogs, just with a more modern colorway.
